css/style.css

```diff
@@ -16,4 +17,285 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ .header, .articles,
+ .footer {
+   max-width: 1200px;
+   margin: 0 auto;
+   font-family: Arial, Helvetica, sans-serif;
+ }
 
+/* Header section  */
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 1rem;
+    padding:1em;
+}
+
+.headerLogo{
+    height:50px;
+    width: auto;
+}
+/* Navbar  */
+
+/* Hamburger menu will not show on a bigger screen size, will only show on smaller device */
+.hamburgerMenu{
+    display: none;
+    width: 30px;
+    height: 30px;
+}
+
+/* Main navbar  that will be visible on bigger device like desktop*/
+.navigation-list{
+    display: flex;
+    list-style: none;
+}
+.navigation-item{
+    padding:0 5px;
+    margin: 0 10px;
+}
+.navigation-link,a{
+    text-decoration: none;
+    color:gray;
+    font-weight: 600;
+    font-size: large;
+}
+.active{
+    color:black;
+}
+.navigation-link:hover{
+    color:rgb(241, 111, 3);
+}
+/* Main content  */
+.content{
+    font-family: Arial, Helvetica, sans-serif;
+}
+    
+ .main{
+    background-image: url(../img/first-background.jpg);
+    background-position:100%;  /* This is important too for the small and big devices adjustment*/
+    background-size: cover; /* Image background-size set to cover for all diveces adjustment */
+    height:85vh; 
+    display: flex; 
+    flex-direction:column;
+    align-items: center;
+    justify-content: center;
+    color:white;
+    font-weight: 100;
+ }
+
+/* Main content's texts  */
+ .main>h1{
+     font-size: 3.5em;
+     letter-spacing:3px;
+ }
+ .h1Paragraph{
```

Comment:
Try to keep the naming style of classes consistent. Usually for css classes people used what's called `kebab-case` (which you have done below).